src/api/cvc4cpp.h

```diff
@@ -1890,9 +1890,18 @@ class CVC4_PUBLIC Solver
                const std::vector<Term>& terms) const;
 
   /* .................................................................... */
-  /* Create Operator Terms                                                */
+  /* Create Operators                                                     */
   /* .................................................................... */
 
+  /**
+   * Create an operator for a builtin Kind
```

Comment:
Can you add a line that the kind may not be an indexed kind?

Reply:
Good idea! Updated with **fc9989c8e73989b41e6f4c030c8ada23a9357283**